packages/plugins/type-designer/src/public/type-designer.simple.scd

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SCL version="2007" revision="B" release="4" xmlns:td="http://localhost:53515/type-designer.schema.xsd">
+<Header id="Type Designer Plugin simple test file"/>
+<DataTypeTemplates>
+	<Private name="Type designer Plugin">
+		<td:SubstationTypesRoot>
+			<td:SubstationType id="substation-type-1" name="Substation Type One" desc="Substation Type One description">
+				<td:VoltageLevelTypeRef to="voltage-level-type-1" />
+			</td:SubstationType>
+		</td:SubstationTypesRoot>
+
+		<td:VoltageLevelTypesRoot>
+			<td:VoltageLevelType id="voltage-level-type-1" name="Voltage Level Type One" desc="Voltage Level Type One description" voltage="" stages="" frequency="">
+				<td:BayTypeRef to="bay-type-1" />
+			</td:VoltageLevelType>
+		</td:VoltageLevelTypesRoot>
+
+		<td:BayTypesRoot>
+			<td:BayType id="bay-type-1" name="Bay Type One" desc="Bay Type One description" type="">
+				<td:IEDTypeRef to="ied-type-1" />
+			</td:BayType>
+		</td:BayTypesRoot>
+
+		<td:IEDTypesRoot>	
+			<td:IEDType id="ied-type-1" name="IED Type One" desc="IED Type One description" type="">
+				<td:LogicalDeviceTypeRef to="ld-type-1" />
+			</td:IEDType>
+		</td:IEDTypesRoot>	
+
+		<td:LogicalDeviceTypesRoot>
+			<td:LogicalDeviceType id="ld-type-1" name="LD Type One" desc="LD Type One description" function="" stage="">
+				<td:LogicalNodeTypeRef to="ln-type-1" />
+			</td:LogicalDeviceType>
+		<td:LogicalDeviceTypesRoot>
+
+		<td:LogicalNodeTypesRoot>
+			<td:LogicalNodeType id="ln-type-1" name="LN Type One" desc="LN Type One description"/>
+		<td:LogicalNodeTypesRoot>
```

Comment:
Logical node types already exist, you can find them as `<LNodeType id="...">` in the SCD

Reply:
I'll change `LogicalNodeType` to `LNodeType`, and in the same way change `LogicialDeviceType` to `LDeviceType`

Reply:
we would have to look outside of our Private field to search for these `<LNodeType>`. Are they the same as what we are building through the type designer ?

Reply:
Pretty much the same, take a look at some examples https://github.com/openscd/open-scd/blob/main/packages/plugins/test/testfiles/wizards/ied.scd?plain=1#L499-L574

Reply:
Ok, Thanks. I see here that these fields are part of the official IEC 61850-6 standard as listed [here](https://en.wikipedia.org/wiki/Substation_Configuration_Language). Do we have [a copy of the standard ](https://www.vde-verlag.de/iec-normen/preview-pdf/info_iec61850-6%7Bed2.1%7Den.pdf)?

Reply:
I dont think we have a copy, but you can ask Philip about it